esx_service/utils/auth_api.py

```diff
@@ -488,9 +488,26 @@ def vm_in_any_tenant(vms):
 
     return None
 
+
+def named_tenant(func):
+
+    def not_supported():
+        error_info = error_code.generate_error_info(ErrorCode.FEATURE_NOT_SUPPORT,
+                                                    auth_data_const.DEFAULT_TENANT)
+        return error_info
+
+    def wrap_function(name, vm_list):
```

Comment:
I am little confused - why do you need a wrap_function?  don't you get *args, **kwargs in the call to decorator ?